### ISSUE TITLE
Nested span tags block forms mode for sliders in Firefox

### ISSUE THREAD

**Issue by chrisauclair** [nvaccess/nvda#5837]
If a `div` with a `role` of `"slider"` contains nested `span` tags, forms mode cannot be activated in Firefox. For example, this code works in all browser except for Firefox:

```
<div tabindex="0" role="slider" aria-valuenow="0.0" aria-valuemin="0" aria-valuemax="100" aria-label="seek bar" aria-valuetext="0:00">
    <div class="slider-text">
        <span class="text"><span>Loaded</span>: 0%</span>
    </div>
</div>
```

Forms mode works if the inner `span` tag is removed, but that's perfectly valid HTML.

This bug was discovered while doing accessibility testing on [video.js](http://videojs.com/).

**Comment by ehollig** [comment]
Any thoughts on this fairly old ticket @jcsteh

**Comment by jcsteh** [comment]
It's debatable as to whether Firefox or NVDA is the more appropriate place to fix this. The fix is trivial in either case. I'm going to try to get it fixed in Firefox, so I filed [Mozilla bug 1455511](https://bugzilla.mozilla.org/show_bug.cgi?id=1455511).

**Comment by jcsteh** [comment]
I fixed this for Firefox 61.
